build/test/integration/multilocale_test.js

```diff
@@ -2,22 +2,22 @@
 
 var assert = require('chai').assert;
 var path = require('path');
-var AdmZip = require('adm-zip');
+var fs = require('fs');
 var helper = require('./helper');
 
 suite('Multilocale integration tests', function() {
   var localesDir = 'build/test/resources/locales';
-  var localesFileObj = {'en-US': '', 'zh-CN': ''};
+  // var localesFileObj = {'en-US': '', 'zh-CN': ''};
 
   suiteSetup(helper.cleanupWorkspace);
   teardown(helper.cleanupWorkspace);
 
   function makeHelper(localesFilePath, localesDir, concat, done) {
-    var settingsZipPath = path.join(process.cwd(), 'profile', 'webapps',
-      'settings.gaiamobile.org', 'application.zip');
-    var cnPathInZip = 'locales-obj/index.zh-CN.json';
+    var settingsFolderPath = path.join(process.cwd(), 'profile', 'apps',
+      'settings');
+    var cnPathInFolder = 'locales-obj/index.zh-CN.json';
     var cnSettingsProperties = 'locales/settings.zh-CN.properties';
-    var langPathInZip = 'shared/resources/languages.json';
+    // var langPathInFolder = 'shared/resources/languages.json';
```

Comment:
I prefer to keep them since they are needed to reenable the assert I disabled because of bug 1268477